ui/raidboss/data/04-sb/trial/rathalos-ex.js

```diff
@@ -4,27 +4,36 @@
 
 // Rathalos Extreme
 [{
-  zoneRegex: /^(The Great Hunt \(Extreme\)|Unknown Zone \(2Fa\))$/,
+  zoneRegex: /^The Great Hunt \(Extreme\)/,
```

Comment:
This lost the ending `$` during an intermediate commit.

Reply:
good spotting. Its now back where it belongs